daily-coding-challlenges/20-05-2020/lastIndexOf.js

```diff
@@ -5,6 +5,14 @@
  * The function should take in an array and a search value and work as intended.
  */
 
-const myLastIndexOf = (arr, searchVal) => {
-	/* Your implementation goes here */
-}
\ No newline at end of file
+
+// const myLastIndexOf = (arr, searchVal) => {
+// 	/* Your implementation goes here */
+// }
+
+let arr = "i am fortune ikechi.";
```

Comment:
@iamfortune, based on our conversation please rework this logic and move the implementation inside the function.